examples/cell/src/index.ts

```diff
@@ -139,7 +139,11 @@ function main(): void {
   sessionContext.kernelChanged.connect(() => {
     void sessionContext.session?.kernel?.info.then(info => {
       const lang = info.language_info;
-      const mimeType = mimeService.getMimeTypeByLanguage(lang);
+      const mimeTypes = mimeService.getMimeTypeByLanguage(lang);
+      const mimeType =
+        Array.isArray(mimeTypes) && mimeTypes.length !== 0
+          ? mimeTypes[0]
+          : mimeTypes;
```

Comment:
Do we need a better fallback if `mimeTypes === []`?

The current code results in:

mimeTypes | mimeType 
--- | ---
 'type1' |  'type1'
['type1', ...] | 'type1'
[] | []

But `CodeEditor.Model.mimeType` should be a string. So should we switch to 

```suggestion
        Array.isArray(mimeTypes)
          ? (mimeTypes[0] ?? IEditorMimeTypeService.defaultMimeType)
          : mimeTypes;
```

Reply:
What if instead we ensured that `getMimeTypeByLanguage()` never returns an empty array?

Reply:
Yes it will ease usage avoiding testing for each call